### ISSUE TITLE
User Story

### ISSUE THREAD

**Issue by MekoHong** [department-of-veterans-affairs/va-mobile-app#6804]
## Description 

**Problem:**

Veterans may not understand the app’s usage of “VA prescriptions.” Many of those non-VA meds are prescribed by VA Providers and sent to non-VA pharmacies so they are technically VA prescriptions (just not dispensed by VA)

**How might we** help Veterans know that medications documented but not dispensed by a VA pharmacy is not listed in app and help provide clarity about what the app's default list include/excludes  

As part of this story we want to:   

- Update the body copy on the list view
- Add new & update some bullets in the help panel

## Acceptance Criteria

  

1. -  [ ] Update the body copy on the list view to now read:  This list only shows prescriptions filled by VA pharmacies and may not include all your medications. See attached screen shots, below

2. -  [ ] Update the help large panel content to now show the following bulleted information

- Bullet 1: New prescriptions not yet processed by a VA pharmacy
- Bullet 2: Prescriptions filled at non-VA pharmacies
- Bullet 3: Prescriptions that expired or discontinued in the last 6 months
- Bullet 4: Medications administered at a clinic or ER (existing)
- Bullet 5: Self-entered medications (existing)

See screenshot in comments for reference 

**Accessibility Requirements**
- N/A

## Notes & Open Questions

- Dependencies/Roadblocks:
    - Any internal/external dependencies?
 
- Test Account: Karsyn

    
- Dev Notes: 

## Ticket Checklist

- [X] Acceptance criteria defined
- [X] Labels added (front-end, back-end, feature)
- [X] Linked to an Epic

**Comment by MekoHong** [comment]
AC 1: [Figma](https://www.figma.com/file/Si4LF4t8VOv8emdCovv3U3/%F0%9F%94%8D-Prescriptions-2.0---Working---VAMobile?type=design&node-id=1288-9199&mode=design&t=zWgXQwJHrSCOem4Q-4)

Screenshot: 
<img width="277" alt="Screenshot 2023-09-22 at 10 13 00 AM" src="https://github.com/department-of-veterans-affairs/va-mobile-app/assets/90273080/57752ea1-5e11-43a4-9ee8-7924d563bd71">

**Comment by MekoHong** [comment]
AC 2: [Figma](http://this%20list%20only%20shows%20prescriptions%20filled%20by%20va%20pharmacies%20and%20may%20not%20include%20all%20your%20medications./)

<img width="249" alt="Screenshot 2023-09-22 at 10 14 11 AM" src="https://github.com/department-of-veterans-affairs/va-mobile-app/assets/90273080/de13bc9c-e4bd-467a-a0e4-b692f76dcc61">

**Comment by MekoHong** [comment]
@mistymg @rachelhanster here is the engineering implementation for the content update for problem D.  @rachelhanster let us know if you have any questions or concerns.  Thanks!

**Comment by rachelhanster** [comment]
Let's have consistency in the app and easy understanding.  In "Help," prescriptions that expired or discontinued: Is it 180 days or 6 months? In the app's latest product guide, I see this language used when veterans don't have a prescription list displayed: "Prescriptions that are inactive for more than 180 days"

**Comment by mistymg** [comment]
My apologies, @rachelhanster! You called this out in the Slack thread and I forgot. Yes, we should be consistent. And the one in the blue alert (when Veterans don't have an Rx list) is more accurate. So we'll go with that.

**Comment by MekoHong** [comment]
Makes sense to include the update with this ticket as well.  I will update the ACs to include

**Comment by MekoHong** [comment]
@rbontrager can you please add a QA estimate for this ticket.  Let me know if you have any questions on the intent.  Thanks!
